daml-assistant/daml-helper/src/DA/Daml/Helper/Start.hs

```diff
@@ -286,15 +289,16 @@ runStart
               forever $ do
                 printRebuildInstructions
                 c <- getChar
-                when (c == 'r' || c == 'R') $ rebuild projectConfig darPath
+                when (c == 'r' || c == 'R') $ rebuild projectConfig darPath sandboxPort runInitScript
                 threadDelay 1000000
-
-        rebuild projectConfig darPath = do
-          putStrLn "re-building and uploading package"
+        rebuild :: ProjectConfig -> FilePath -> SandboxPort -> IO () -> IO ()
+        rebuild projectConfig darPath sandboxPort doRunInitScript = do
+          putStrLn "Re-building and uploading package."
```

Comment:
I like it to end such lines with `...` to signal that something is going on. But's that obviously a matter of personal taste.

Reply:
added some dots ...